packages/svelteui-core/src/styles/theme/SvelteUIProvider/SvelteUIProvider.svelte

```diff
@@ -36,7 +36,7 @@
 	const useStyles = createStyles(() => ({ root: {} }));
 	const forwardEvents = createEventForwarder(get_current_component());
 	const DEFAULT_THEME = useSvelteUITheme();
-	const currentTheme = () => {
+	$: currentTheme = () => {
```

Comment:
Since we are changing this, lets change this variable to an object instead of a function, it makes no sense to be a function.

```suggestion
let currentTheme = null;
$: {
  if (themeObserver === null) currentTheme = null;
  currentTheme = themeObserver === 'light' ? (mergedTheme as unknown as string) : (dark as string);
};
```

(also change below when it's being used)

Reply:
Done